From 3cc6eeb2df0a9d6511317ea71059e455e04d2a0e Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Tue, 1 Jul 2025 11:33:35 -0400
Subject: [PATCH] Don't try to use the file transfer portal on OSX

Make the ifdefs here match what we use in gtk4.

Fixes: #7630
---
 gtk/gtkselection.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git gtk/gtkselection.c gtk/gtkselection.c
index bb687755617..b4f35ef840a 100644
--- gtk/gtkselection.c
+++ gtk/gtkselection.c
@@ -526,7 +526,7 @@ gtk_target_list_add_uri_targets (GtkTargetList *list,
 
   gtk_target_list_add (list, text_uri_list_atom, 0, info);
 
-#ifndef G_OS_WIN32
+#if defined(G_OS_UNIX) && !defined(__APPLE__)
   if (file_transfer_portal_supported ())
     {
       gtk_target_list_add (list, portal_filetransfer_atom, 0, info);
-- 
GitLab

